src/ophys_etl/modules/video/utils.py

```diff
@@ -624,19 +695,54 @@ def _video_array_from_h5(
     if reticle:
         for ii in range(d_reticle, video_shape[1], d_reticle):
             old_vals = np.copy(video_as_uint[:, ii:ii+2, :, :])
-            new_vals = np.zeros(old_vals.shape, dtype=np.uint8)
-            new_vals[:, :, :, 0] = 255
-            new_vals = (new_vals//2) + (old_vals//2)
-            new_vals = new_vals.astype(np.uint8)
+            new_vals = np.zeros(old_vals.shape, dtype=video_dtype)
+            new_vals[:, :, :, 0] = max_cast_value
+            new_vals = (new_vals//4) + (3*old_vals//4)
+            new_vals = new_vals.astype(video_dtype)
             video_as_uint[:, ii:ii+2, :, :] = new_vals
         for ii in range(d_reticle, video_shape[2], d_reticle):
             old_vals = np.copy(video_as_uint[:, :, ii:ii+2, :])
-            new_vals = np.zeros(old_vals.shape, dtype=np.uint8)
-            new_vals[:, :, :, 0] = 255
-            new_vals = (new_vals//2) + (old_vals//2)
-            new_vals = new_vals.astype(np.uint8)
+            new_vals = np.zeros(old_vals.shape, dtype=video_dtype)
+            new_vals[:, :, :, 0] = max_cast_value
+            new_vals = (new_vals//4) + (3*old_vals//4)
+            new_vals = new_vals.astype(video_dtype)
             video_as_uint[:, :, ii:ii+2, :] = new_vals
 
         logger.info('added reticles')
 
     return video_as_uint
+
+
+def apply_mean_filter_to_video(
+        video: np.ndarray,
+        kernel_size: int) -> np.ndarray:
+    """
+    Use skimage.measure.block_reduce to downsample a video
+    spatially by taking the mean of (kernel_size, kernel_size)
+    blocks
+
+    Parameters
+    ----------
+    video: np.ndarray
+        (ntime, nrows, ncols)
+
+    kernel_size: int
```

Comment:
Sorry, but can you please mention that the kernel is square?

Reply:
The first sentence of the docstring says
```
    Use skimage.measure.block_reduce to downsample a video
    spatially by taking the mean of (kernel_size, kernel_size)
    blocks
```
Is specifying the `(kernel_size, kernel_size)` shape of the blocks not enough (or is the notation confusing)?

Reply:
I was looking at the parameter description and missed the function description. It is clear, unless you want to add a note to the `kernel_size` description that it is square.